WooCommerce/Classes/ViewModels/Order Details/OrderPaymentDetailsViewModel.swift

```diff
@@ -155,8 +155,8 @@ final class OrderPaymentDetailsViewModel {
             return nil
         }
 
-        // We want the condensed refund total because it's reported as a negative value
-        return currencyFormatter.formatAmount(condensedRefund.total, with: order.currency)
+        // We can not assume the total is negative.
+        return currencyFormatter.formatAmount(condensedRefund.normalizedTotalAsNegative, with: order.currency)
```

Comment:
now that we don't create `OrderRefundCondensed` with a positive amount, do you think we still want to normalize the amount to negative? I tested with just removing `handleCondensedRefundsIfNeeded` and that seems enough 🙂

Reply:
I guess it doesn't hurt to be extra safe?